documentation/modules/post/multi/gather/irssi_creds.md

```diff
@@ -0,0 +1,27 @@
+## Vulnerable Application
+
+  Any system with a `shell` or `meterpreter` session.
```

Comment:
the point of doing this is so that in 3yrs when it doesn't work anymore, we have a quick reference to show a known good version (linking to the installer is a bonus), and is more or less derived from the PR anyways